waspc/data/Generator/templates/react-app/src/queries/index.d.ts

```diff
@@ -1,14 +1,10 @@
 import { UseQueryResult } from "@tanstack/react-query";
 
-import { type HttpMethod } from "../types";
-
 export type Query<Input, Output> = {
-    (args: Input): Promise<Output>
-    queryCacheKey: string[]
-    route: { method: HttpMethod, path: string }
+    (queryCacheKey: string[], args: Input): Promise<Output>
 }
 
-export function useQuery<Input, Output, Error = unknown>(
+export function useQuery<Input, Output>(
```

Comment:
We always throw something that resembles a general `Error`, so let's keep it simple for now.